test/JDBC/expected/single_db/db_accessadmin-vu-verify.out

```diff
@@ -0,0 +1,337 @@
+
+
+-- tsql
+-- CREATE LOGIN AND ADD IT TO db_accessadmin
+    -- Only members of db_owner should be able to add/drop members to db_accessadmin
+-- db_accessadmin should be able to do the following
+    -- CREATE USER in database
+    -- DROP ANY USER in database (except dbo)
+    -- ALTER default_schema for any user in database
+    -- RENAME user execpt for members of db_owner
+    -- CREATE SCHEMA for self or other users in database
+-- bbf dump does not dump password so reset the password
+ALTER LOGIN babel_5136_l1 WITH PASSWORD = '12345678'
+GO
+-- CREATE A USER WITH db_accessadmin privilege in DATABASE babel_5136
+USE master
+GO
+CREATE LOGIN babel_5136_db_accessadmin_l1 WITH PASSWORD = '12345678'
+GO
+CREATE USER babel_5136_user_master FOR LOGIN babel_5136_db_accessadmin_l1
+GO
+USE babel_5136
+GO
+CREATE USER babel_5136_db_accessadmin_user FOR LOGIN babel_5136_db_accessadmin_l1
+GO
+ALTER ROLE db_accessadmin ADD MEMBER babel_5136_db_accessadmin_user
+GO
+-- Cannot add login as member of db_accessadmin
+ALTER ROLE db_accessadmin ADD MEMBER babel_5136_l1
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: role "babel_5136_babel_5136_l1" does not exist)~~
+
+-- MEMBERS OF db_accessadmin WILL ALWAYS HAVE CONNECT PRIVILEGES
+REVOKE CONNECT FROM babel_5136_db_accessadmin_user
+GO
+-- terminate-tsql-conn
+
+-- tsql user=babel_5136_l1 password=12345678 database=babel_5136
+-- Only db_owner members should be able to do this
+ALTER ROLE db_accessadmin ADD MEMBER babel_5136_db_accessadmin_user
+GO
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Cannot alter the role 'db_accessadmin', because it does not exist or you do not have permission.)~~
```

Comment:
Let's add positive test case for this explicitly.

Reply:
Already added, also added a comment above that line for easier test case review